.github/workflows/vast.yaml

```diff
@@ -1079,23 +607,448 @@ jobs:
+          # The asset names are constant so we can permanently link to
+          # https://github.com/tenzir/vast/releases/latest/download/vast-debian-release-gcc.tar.gz
+          # https://github.com/tenzir/vast/releases/latest/download/vast-debian-release-clang.tar.gz
+          # for builds of the latest release.
+          asset_name: "${{ env.PUBLISH_NAME }}.tar.gz"
+          asset_content_type: application/gzip
+
+      - name: Publish VAST.spdx to GitHub Release
+        if: ${{ github.event_name == 'release' && matrix.vast.name == 'Debian' }}
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: "VAST.spdx"
+          asset_name: "VAST.spdx"
+          asset_content_type: text/plain
+
+  vast-plugins:
+    needs:
+      - vast
+      - configure
+    if: ${{ always() && needs.configure.outputs.run-vast-plugins == 'true' }}
+    runs-on: ${{ matrix.plugin.os }}
+    container: ${{ matrix.plugin.container }}
+    strategy:
+      fail-fast: false
+      matrix:
+        plugin:
+          - name: Example Analyzer Debian
+            target: example-analyzer
+            path: examples/plugins/analyzer
+            dependencies:
+            os: ubuntu-20.04
+            container: debian:bullseye-slim
+          - name: Example Analyzer macOS
+            target: example-analyzer
+            path: examples/plugins/analyzer
+            dependencies:
+            os: macos-latest
+            container:
+          - name: Example Pipeline Operator Debian
+            target: example-pipeline-operator
+            path: examples/plugins/pipeline_operator
+            dependencies:
+            os: ubuntu-20.04
+            container: debian:bullseye-slim
+          - name: Example Pipeline Operator macOS
+            target: example-pipeline-operator
+            path: examples/plugins/pipeline_operator
+            dependencies:
+            os: macos-latest
+            container:
+          - name: Broker Debian
+            target: broker
+            path: plugins/broker
+            dependencies:
+              - libbroker-dev
+            os: ubuntu-20.04
+            container: debian:bullseye-slim
+          - name: Sigma Debian
+            target: sigma
+            path: plugins/sigma
+            os: ubuntu-20.04
+            container: debian:bullseye-slim
+          - name: Sigma macOS
+            target: sigma
+            path: plugins/sigma
+            os: macos-latest
+            container:
+          - name: Parquet Debian
+            target: parquet
+            path: plugins/parquet
+            os: ubuntu-20.04
+            container: debian:bullseye-slim
+          - name: Parquet macOS
+            target: parquet
+            path: plugins/parquet
+            os: macos-latest
+            container:
+          - name: Web Debian
```

Comment:
If I'm not mistaken then the PCAP plugin is missing from this list.

Reply:
Seems to be missing from debian as well. I guess it should be added for both, rgiht?

Reply:
Yes.